corehq/util/urlsanitize/urlsanitize.py

```diff
@@ -19,12 +17,9 @@ def sanitize_user_input_url(url):
         raise InvalidURL()
     if scheme not in ['http', 'https']:
         raise PossibleSSRFAttempt('scheme not http(s)')
-    try:
-        ip_address_text = resolve_ip(hostname)
-    except socket.gaierror:
-        raise CannotResolveHost()
-    ip_address = ipaddress.ip_address(ip_address_text)
-    return sanitize_ip(ip_address)
+
+    for ip_address in resolve_to_ips(hostname):
+        sanitize_ip(ip_address)
```

Comment:
This method is documented to `Return ip address if the url is safe`, but it doesn't return anything. It should either return a value or the docstring should be changed. If the docstring is changed it may also be worth renaming the function to `validate_user_input_url` since "sanitize" implies producing something new/clean.

Reply:
I agree -- renaming away from 'sanitize' was on my to-do list, but thanks for pointing out the inconsistency with the comment